test/libgit2.jl

```diff
@@ -57,6 +57,14 @@ end
     @test sig3.email == sig.email
 end
 
+@testset "Default config" begin
+    #for now just see that this works
+    cfg = LibGit2.GitConfig()
+    @test isa(cfg, LibGit2.GitConfig)
```

Comment:
I guess it doesn't hurt having this test, but couldn't one safely assume that the `LibGit2.GitConfig` constructor returns a `LibGit2.GitConfig` object? What else could you get?